test/unit/imgix-provider.test.jsx

```diff
@@ -66,6 +83,7 @@ describe("ImgixProvider", () => {
     const wrappedComponent = (
       <ImgixProvider {...providerProps}>
         <ReactImgix {...modifiedProps} />
+        <Background {...modifiedProps} />
```

Comment:
question: Is the `should merge the Provider and Child props` testing the `Background` props, or only the `ReactImgix` props? Following along with the comments:

```
renderedComponent
      .childAt(0) // mergePropsHOF
      .childAt(0) // processPropsHOF
      .childAt(0) // shouldComponentUpdateHOC
      .childAt(0) // ChildComponent
      .props();
```

It seems to be explicitly testing the first child, `.childAt(0) // ChildComponent`, IE `ReactImgix`.

Reply:
Fixed, thanks!